app/src/main/java/com/futurice/freesound/common/utils/Parcels.kt

```diff
@@ -0,0 +1,92 @@
+
+package com.futurice.freesound.common.utils
+
+import android.os.Parcel
+import android.os.Parcelable
+import java.math.BigDecimal
+import java.math.BigInteger
+import java.util.*
+import kotlin.collections.ArrayList
+
+interface KParcelable : Parcelable {
```

Comment:
For that, we would need espresso, but I cannot see it in the project (?!). If that is the case, after this PR I will add espresso